.github/workflows/flink_cdc_base.yml

```diff
@@ -148,7 +149,7 @@ jobs:
           maven-version: 3.8.6
 
       - name: Compile and test
-        timeout-minutes: 90
+        timeout-minutes: 60
```

Comment:
[INFO] Running org.apache.flink.cdc.connectors.polardbx.PolardbxCharsetITCase
Error: The action 'Compile and test' has timed out after 60 minutes.

Reply:
`PolardbxCharsetITCase` was started at ~16:27 and timed out eventually at 16:49. Locally this case wouldn't take more than 3 minutes to finish, so it might be a bug that should be closed by #3902.